RELEASE.md

```diff
@@ -1,78 +1,164 @@
-1. [NGINX](https://github.com/kubernetes/ingress-nginx/tree/main/images/nginx)
+## 1. [NGINX Base Image](https://github.com/kubernetes/ingress-nginx/tree/main/images/nginx)
 
-* Open pull request
+A. Open pull request
   
-If you are updating any component in [build.sh](images/nginx/rootfs/build.sh) please also update the SHA256 checksum of that component as 
-  well, the cloud build will fail with an exit 10 if not.  
+  If you are updating any component in [build.sh](images/nginx/rootfs/build.sh) please also update the SHA256 checksum of that component as well, the cloud build will fail with an exit 10 if not.  
 Example [NGINX_VERSION](images/nginx/rootfs/build.sh#L21),
 [SHA256](images/nginx/rootfs/build.sh#L124) 
-* Merge
-* Wait for [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
 
-1a. Promote images:
+B. Merge
 
-Open pull request to promote staging image:
-[add sha - version](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml#L1)
+C. Wait for [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
+
+D. Promote Images:
+
+  Open pull request to promote staging image:
+  [add sha - version](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml#L1)
 
 The sha is available in output from [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
 
-2. Change to images:
+## 2. Change to other Images:
```

Comment:
I think it's not really clear what this section should tell us.

Reply:
Agreed, we should add a blurb about all the images we maintain. There's a new one too for the certs that aren't on this list either.